terraform/system/aws/variables.tf

```diff
@@ -229,6 +229,16 @@ variable "region" {
   default = "us-east-1"
 }
 
+variable "registry_pull_qps" {
```

Comment:
change param names to `kubelet_registry_pull_qps` and `kubelet_registry_burst` everywhere